src/Kraken.h

```diff
@@ -53,6 +53,7 @@ class Kraken {
    void FreeChunk();
 
 private:
+   const std::string kCancel = {"<CANCEL>"};
```

Comment:
I was thinking on it but I rather keep it here for now. This is a one off deal, since there is no other messaging in this "cross stream" direction.

 I could have kept it local inside the function too